src/Storages/ObjectStorageQueue/ObjectStorageQueueSource.cpp

```diff
@@ -578,6 +568,26 @@ void ObjectStorageQueueSource::FileIterator::releaseFinishedBuckets()
     }
 }
 
+std::string ObjectStorageQueueSource::FileIterator::bucketHoldersToString() const
+{
+    std::string processors_infos;
+    for (const auto & [processor, bucket_holder] : bucket_holders)
+    {
+        if (!processors_infos.empty())
+            processors_infos += ", ";
+
+        processors_infos += fmt::format("processor {} -> {} buckets", processor, bucket_holder.empty());
```

Comment:
Was it supposed to be


```suggestion
        processors_infos += fmt::format("processor {} -> {} buckets", processor, bucket_holder.size());
```

?